main.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"fmt"
+	"github.com/vskut/go-temp/strtoint"
+)
+
+func main() {
```

Comment:
it's a package that can skip `main` func. Nothing bad in having it, just keep in mind, `main` is not a mandatory point in every package.

Reply:
Just debugging by old-way